packages/ia-components/package.json

```diff
@@ -21,9 +21,10 @@
         "@storybook/addon-knobs": "^4.1.4",
         "@storybook/react": "^4.1.4",
         "assert": "^1.4.1",
+        "babel-core": "^7.0.0-bridge.0",
```

Comment:
Question: why are we using `babel-core` instead of `@babel/core` v. 7?

Reply:
we already use ` "@babel/core": "^7.2.2",` at the first line, we should not need the two unless it is for a certain reason.

Reply:
@iisa 
Your concern is valid. I will need to check with docs and `babel-upgrade` package which I used for upgrading the version. I will do some research and reply.

Reply:
@iisa I think this happened because `ia-components/package.json` was already using Babel 7 and `babel-upgrade` package was I used was designed to migrate from Babel 6 to Babel 7. So changes in this `package.json` were not required. Thanks for reviewing. 
I will update the PR.

Reply:
Jest JS docs say otherwise.  Perhaps upgrade tool is behind.  Pleaes double check. https://jestjs.io/docs/en/getting-started

![Screen Shot 2019-03-25 at 9 18 15 AM](https://user-images.githubusercontent.com/7840857/54936151-13e59f80-4edf-11e9-98ed-04b635bd4700.png)

Reply:
@iisa I again cross-checked. 
It is strange but `babel-core` is **needed** in case of jestJS.
Relevant links:
1: https://github.com/facebook/jest/issues/6913#issuecomment-417123948
2: https://github.com/facebook/jest/issues/6662#issuecomment-403718906
All the the solutions say the same thing, i.e. keep both @babel/core and babel-core in case of jest as a dependency.
So I strongly think that the `babel-upgrade` is doing the right thing.
Also if you see link 1, removing `babel-core` gives errors.
So concluding on the note that `babel-upgrade` is doing the right thing in my opinion.